.gitlab-ci.yml

```diff
@@ -39,12 +39,21 @@ lint:
     - yarn
     - yarn lint
 
+typecheck:
+  stage: test
+  tags: ['runner:main', 'size:large']
+  image: $CI_IMAGE
+  script:
+    - yarn
+    - yarn typecheck
+
 build:
   stage: test
   tags: ['runner:main', 'size:large']
   image: $CI_IMAGE
   script:
     - yarn
+    - yarn typecheck
```

Comment:
since there is a dedicated typecheck step, do we need to also add it here?

Reply:
Wops that was unintended. Removing it.